test/verify/check-system-info

```diff
@@ -252,7 +252,7 @@ class TestSystemInfo(MachineCase):
         b.click("#systime-apply-button")
         b.wait_popdown("system_information_change_systime")
 
-        b.wait_text("#system_information_systime_button", "2020-01-24 08:03")
+        b.wait_text("#system_information_systime_button", "Jan 24, 2020 8:03 AM")
```

Comment:
Most probably you need to conditionalize this for 8-2-distropkg.

Reply:
of course it will! I thought about it, but of course totally forgot about it...